jina/serve/networking.py

```diff
@@ -181,7 +182,11 @@ async def _init_stubs(self):
             self._initialized = True
 
         async def send_requests(
-            self, requests: List[Request], metadata, compression
+            self,
+            requests: List[Request],
+            metadata,
+            compression,
+            timeout: Optional[float] = 1.0,
```

Comment:
let's have the same default in the library as in the parsers for sanity and clarity